modules/InternalPropTypes.js

```diff
@@ -14,7 +14,7 @@ export const history = shape({
   goForward: func.isRequired
 })
 
-export const component = oneOfType([ func, string ])
+export const component = oneOfType([ func, string, object, number, symbol ])
```

Comment:
Should probably just use `PropTypes.elementType`?

Reply:
LGTM 👍